modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -735,6 +734,10 @@ public static boolean isDecimalSchema(Schema schema) {
                 && "number".equals(schema.getFormat());
     }
 
+    public static boolean isValidEnumSchema(Schema schema) {
+        return schema.getEnum() != null && !schema.getEnum().isEmpty();
+    }
+
```

Comment:
can you please merge the latest master into your branch and use `isEnumSchema` instead (added recenlty)?

Reply:
Done in https://github.com/OpenAPITools/openapi-generator/pull/18623/commits/a26013824d60e2222e5c976573295853e4562b3f

Reply:
thanks for the prompt update. will merge if all tests pass

thanks again for the PR